assets/src/components/ladderPage.tsx

```diff
@@ -92,7 +92,7 @@ const LadderTab = ({
         {tab.isCurrentTab && showSaveIcon ? (
           <button
             className="m-ladder-page__tab-save-button"
-            data-testid="save-button"
+            title="save"
```

Comment:
We've usually used title case for the `title` attribute since that looks better in the tooltip. I'm not sure if that's a hard and fast rule, but it's at least the convention I've been following so far.